### ISSUE TITLE
Adjust relative time (or UTC time) based on remote time

### ISSUE THREAD

**Issue by oxygen** [moment/moment#835]
(I have just finished writing a extremely small js class similar to some moment.js functionality.
...if only moment.js had shown up on the previous searches :( I would have saved a few hours.)

One of the features I needed (and implemented) was to adjust relative times based on server time (obtained at page render, or via RPC), because of a few number of users with grossly incorrect timezones (which usually results in incorrect UTC timestamps (as seen by the user)).

Basically, the UTC time is adjusted with n seconds before beeing used anywhere.

Does moment.js support this? 
If not, could it be added (I am willing to do it, as my time allows it)?
